app/controllers/CreditAndRefundController.scala

```diff
@@ -125,22 +131,21 @@ class CreditAndRefundController @Inject()(val authorisedFunctions: FrontendAutho
       .toList.sortWith((p1, p2) => sortingOrderCreditType(p1._1) < sortingOrderCreditType(p2._1))
       .map {
         case (documentId, credits) => (documentId, sortCredits(credits))
-    }.flatMap {
-        case (_, credits) => credits
+      }.flatMap {
+      case (_, credits) => credits
     }
     creditsGroupedPaymentTypes
   }
 
   def getCreditTypeGroupKey(credits: (DocumentDetailWithDueDate, FinancialDetail)): String = {
-    val isMFA : Boolean = credits._2.validMFACreditType()
-    val isCutOverCredit : Boolean = credits._2.mainType.get == "ITSA Cutover Credits"
-    val isPayment : Boolean = credits._1.documentDetail.paymentLot.isDefined
+    val isMFA: Boolean = credits._2.validMFACreditType()
+    val isCutOverCredit: Boolean = credits._2.mainType.get == "ITSA Cutover Credits"
```

Comment:
I understand this is not your code, but we should use next method moving forward =>
mainType.validCutoverCreditType()

Reply:
will be a part of another story